reference/configuration/framework.rst

```diff
@@ -858,10 +859,136 @@ setting the value to ``null``:
 templating
 ~~~~~~~~~~
 
-.. _reference-framework-assets-version:
+hinclude_default_template
+.........................
+
+**type**: ``string`` **default**: ``null``
+
+Sets the content shown during the loading of the fragment or when JavaScript
+is disabled. This can be either a template name or the content itself.
+
+.. seealso::
+
+    See :ref:`book-templating-hinclude` for more information about hinclude.
+
+.. _reference-templating-cache:
+
+cache
+.....
+
+**type**: ``string``
+
+The path to the cache directory for templates. When this is not set, caching
+is disabled.
+
+.. note::
+
+    When using Twig templating, the caching is already handled by the
+    TwigBundle and doesn't need to be enabled for the FrameworkBundle.
+
+.. _reference-templating-form:
+
+form
+....
+
+resources
+"""""""""
+
+**type**: ``string[]`` **default**: ``['FrameworkBundle:Form']``
+
+A list of all resources for form theming in PHP. This setting is not required
+if you're using the Twig format for your templates, in that case refer to
+:ref:`the form book chapter <book-forms-theming-twig>`.
+
+Assume you have custom global form themes in
+``src/WebsiteBundle/Resources/views/Form``, you can configure this like:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        framework:
+            templating:
+                form:
+                    resources:
+                        - 'WebsiteBundle:Form'
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+
```

Comment:
Oh, I just copy-pasted the XML sample from another section of the documentation, actually I never use any XML config...

Why blank lines ? I guess it doesn't really affect the behaviour of the XML parser, but gains in visibility ?